src/cpp/src/speculative_decoding/continuous_batching_for_speculative_decoding_impl.cpp

```diff
@@ -20,7 +20,6 @@ ContinuousBatchingPipeline::ContinuousBatchingForSpeculativeDecodingImpl::Contin
 
 void
 ContinuousBatchingPipeline::ContinuousBatchingForSpeculativeDecodingImpl::finish_request(SequenceGroup::Ptr request) {
```

Comment:
Minor comment: generations of draft model will be lost, so speculative decoding won't be so good here.. anyway, not a problem at all, because request synchronization and draft_model generation handling is added here: https://github.com/openvinotoolkit/openvino.genai/pull/1056/files